### DISCUSSION TITLE
Line breaks aren't rendered when inside a label's contents

### DISCUSSION THREAD

**Discussion by joeyj** [grafana/grafana#39148] [Observability]
I've had to redact a bit from this screenshot but it highlights that the label contents of stack_trace are not rendering line breaks whereas the line_format result with the same label does. It would be helpful to render line breaks in labels to assist in reading things like stack traces.

![InkedCapture_LI](https://user-images.githubusercontent.com/365289/133118516-b2b26093-352c-41a1-b30f-4f3ba72c1d5a.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
